themes/hive/layouts/partials/features.html

```diff
@@ -0,0 +1,170 @@
+                    <button class="custom-button feature-button-style" role="button">Learn More</button>
+                </a>
+            </div>
+        </div>
+
+        <div class="row divs">
+            <div class="col-md feature-border">
+                <h2 class="topic-text-style">Hive Metastore Server (HMS)</h2>
+                <p>
+                    The Hive Metastore (HMS) is a central repository of metadata for Hive tables and partitions in a relational database,
+                    and provides clients (including Hive, Impala and Spark) access to this information using the metastore service API.
+                    It has become a building block for data lakes that utilize the diverse world of open-source software, such as Apache Spark and Presto.
+                    In fact, a whole ecosystem of tools, open-source and otherwise, are built around the Hive Metastore, some of which this diagram illustrates.
+                </p>
+                <a href="{{ .Site.Params.features.hms }}" class="icon-block">
+                    <button class="custom-button feature-button-style" role="button">Learn More</button>
+                </a>
+            </div>
+            <div class="col-md">
+                <a href="{{ .Site.Params.features.hms }}">
+                    <img src="{{.Site.BaseURL}}/images/HMS.jpg" width="600" height="400"  alt="Apache Software Foundation"></a>
+                </a>
+            </div>
+        </div>
+
+        <div class="row divs">
+            <div class="col-md ">
+                <a href="{{ .Site.Params.features.acidTxn }}">
+                    <img src="{{.Site.BaseURL}}/images/hiveAcid.svg" width="300" height="300"  alt="Apache Software Foundation"></a>
+                </a>
+            </div>
+            <div class="col-md feature-border">
+                <h2 class="topic-text-style">Hive ACID</h2>
+                <p>
+                    Hive provides full acid support for ORC tables out and insert only support to all other formats.
+                </p>
+                <a href="{{ .Site.Params.features.acidTxn }}" class="icon-block">
+                    <button class="custom-button feature-button-style" role="button">Learn More</button>
+                </a>
+            </div>
+        </div>
+
+        <div class="row divs">
+            <div class="col-md feature-border">
+                <h2 class="topic-text-style">Hive Data Compaction</h2>
+                <p>
+                    Query-based and MR-based data compactions are supported out-of-the-box.
+                </p>
+                <a href="{{ .Site.Params.features.compactions }}" class="icon-block">
+                    <button class="custom-button feature-button-style" role="button">Learn More</button>
+                </a>
+            </div>
+            <div class="col-md">
+                {{- partial "compaction.html" . -}}
+            </div>
+        </div>
```

Comment:
This is only relevant (and necessary) for ACID so maybe it is better to merge it with the previous section instead of highlighting as a key feature.

Reply:
Was this comment addressed?

Reply:
Not yet,  I would like to include it in a later patch along with a few other additions mentioned here: https://lists.apache.org/thread/f6f7h4w11bcggom94sovnrm3d22nwb43